pyiron_atomistics/atomistics/structure/analyse.py

```diff
@@ -532,6 +532,40 @@ def pyscal_voronoi_volume(self):
         """    Calculate the Voronoi volume of atoms        """
         return analyse_voronoi_volume(atoms=self._structure)
 
+    def pyscal_find_solids(self, neighbor_method="cutoff",
+        cutoff=0, bonds=0.5,
+        threshold=0.5, avgthreshold=0.6,
+        cluster=False, q=6, right=True,
+        return_sys=False,
+        ):
+        """
+        Get the number of solids or the corresponding pyscal system.
```

Comment:
Typo?

Reply:
what do you mean?

Reply:
Since I misunderstood the method, I thought `or` should be `of` or some such...  I promise to read more carefully next time. ;)